.circleci/config.yml

```diff
@@ -1,14 +1,14 @@
 version: 2
+
 general:
   branches:
     ignore:
       - gh-pages
+
 jobs:
   build:
     docker:
       - image: circleci/node:10-browsers
-    environment:
-      TZ: '/usr/share/zoneinfo/America/Los_Angeles'
```

Comment:
Do we know this is safe?  Not sure why this is here or what the consequences of removing it are.